compiler/src/iree/compiler/API/README.md

```diff
@@ -1,11 +1,4 @@
-# IREE Compiler Backend
+# IREE Compiler API (deprecated)
 
-This is a top-level project for building public facing API packages that
-combine all dependent MLIR-based projects along with IREE's compiler backend
-API.
-
-It exports artifacts:
-
-* `iree-compiler-backend` Python wheel and source distributions, providing
-  the `iree.compiler_backend` Python packages.
-* Compiler C-API source and binary tarballs (future), with some CLI tools.
+This directory is being refactored into API2. Currently, it only contains
```

Comment:
I think we should probably still leave the documentation until it's actually deleted.

Reply:
> I think we should probably still leave the documentation until it's actually deleted.

Was my thought too. Until I read it and realized it was hopelessly wrong and better deleted.